src/praisonai/praisonai/adapters/__init__.py

```
@@ -44,7 +71,7 @@ def __getattr__(name: str):
 
 def __dir__():
     """List available attributes."""
-    return list(_LAZY_IMPORTS.keys())
+    return list(_LAZY_IMPORTS.keys()) + sorted(_LOCAL_EXPORTS)
 
 
-__all__ = list(_LAZY_IMPORTS.keys())
+__all__ = list(_LAZY_IMPORTS.keys()) + sorted(_LOCAL_EXPORTS)
```

Ruff (0.15.20)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)